### PR TITLE
GetNamespace extension method removed in favor of Type.Namespace

### PR DESCRIPTION
**Pull request by amithegde**
resolving #60

### CODE REVIEW

**Comment by amithegde** [comment]
All unit tests pass

File: src/Blazor.Fluxor/DependencyInjection/DependencyScanners/MiddlewareScanner.cs [hunk 1]
```diff
@@ -1,17 +1,16 @@
-﻿using Blazor.Fluxor.Extensions;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.Linq;
 using System.Reflection;
 
 namespace Blazor.Fluxor.DependencyInjection.DependencyScanners
 {
-	internal class MiddlewareScanner
+    internal class MiddlewareScanner
```
Comment (mrpmorris) [inline src/Blazor.Fluxor/DependencyInjection/DependencyScanners/MiddlewareScanner.cs]: Could you please indent using tabs

File: src/Blazor.Fluxor/DependencyInjection/ServiceCollectionExtensions.cs [hunk 2]
```diff
@@ -8,10 +7,10 @@
...
 namespace Blazor.Fluxor
 {
-	/// <summary>
-	/// Extensions for <see cref="IServiceCollection"/>
-	/// </summary>
-	public static class ServiceCollectionExtensions
+    /// <summary>
+    /// Extensions for <see cref="IServiceCollection"/>
+    /// </summary>
+    public static class ServiceCollectionExtensions
```
Comment (mrpmorris) [inline src/Blazor.Fluxor/DependencyInjection/ServiceCollectionExtensions.cs]: Please indent with tabs

File: src/Blazor.Fluxor/DependencyInjection/DependencyScanners/MiddlewareScanner.cs (hunk 1 above)
Comment (amithegde) [reply src/Blazor.Fluxor/DependencyInjection/DependencyScanners/MiddlewareScanner.cs]: Sure, will send an iteration in the evening.

**Comment by mrpmorris** [comment]
I'd really like to get this merged in for the next release. Any chance you could fix the whitespace please?

**Comment by amithegde** [comment]
> I'd really like to get this merged in for the next release. Any chance you could fix the whitespace please?

done

File: src/Blazor.Fluxor/DependencyInjection/ServiceCollectionExtensions.cs (hunk 2 above)
Comment (amithegde) [reply src/Blazor.Fluxor/DependencyInjection/ServiceCollectionExtensions.cs]: done

**Pull request merged**